[functorch] Update notebooks for latest release

zou3519 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #85856
* __->__ #85855
* #85854

This PR:
- dedups our colab notebooks with the regular functorch notebooks. The
colab notebooks were versions of the reuglar notebooks that had
install instructions. Now that functorch is easier to install, we do not
need those anymore.
- fixes the colab links

Test Plan:
- build docs locally and tested them

zou3519 committed:
[functorch] Update notebooks for latest release

This PR:
- dedups our colab notebooks with the regular functorch notebooks. The
colab notebooks were versions of the reuglar notebooks that had
install instructions. Now that functorch is easier to install, we do not
need those anymore.
- fixes the colab links

Test Plan:
- build docs locally and tested them

[ghstack-poisoned]

zou3519 committed:
Update on "[functorch] Update notebooks for latest release"


This PR:
- dedups our colab notebooks with the regular functorch notebooks. The
colab notebooks were versions of the reuglar notebooks that had
install instructions. Now that functorch is easier to install, we do not
need those anymore.
- fixes the colab links

Test Plan:
- build docs locally and tested them

[ghstack-poisoned]

zou3519 committed:
Update on "[functorch] Update notebooks for latest release"


This PR:
- dedups our colab notebooks with the regular functorch notebooks. The
colab notebooks were versions of the reuglar notebooks that had
install instructions. Now that functorch is easier to install, we do not
need those anymore.
- fixes the colab links

Test Plan:
- build docs locally and tested them

[ghstack-poisoned]

samdow reviewed (approved):
LGTM! As a question vaguely related to this PR, have we done a sanity check that all of the examples still work (I don't think we've changed anything too drastic but might be good to just check)?

zou3519 commented:
> have we done a sanity check that all of the examples still work (I don't think we've changed anything too drastic but might be good to just check)?

We have not, it would be good to check yes

zou3519 committed:
Update on "[functorch] Update notebooks for latest release"


This PR:
- dedups our colab notebooks with the regular functorch notebooks. The
colab notebooks were versions of the reuglar notebooks that had
install instructions. Now that functorch is easier to install, we do not
need those anymore.
- fixes the colab links

Test Plan:
- build docs locally and tested them

[ghstack-poisoned]
